tools/perfscale-load-generator/flags/flags.go

```diff
@@ -41,7 +41,7 @@ func init() {
 
 	flag.StringVar(&Kubeconfig, "kubeconfig", "", "absolute path to the kubeconfig file")
 	flag.StringVar(&Kubemaster, "master", "", "kubernetes master url")
-	flag.IntVar(&Verbosity, "v", 2, "log level for V logs")
+	flag.IntVar(&Verbosity, "verbose", 2, "log level for V logs")
```

Comment:
nit: can you also change the usage in the readme.md to reflect the change? https://github.com/kubevirt/kubevirt/blob/1ed2f70e434e78ec53ab7e4571e75f7774f5feb5/tools/perfscale-load-generator/README.md